rel/apps/couch_epi.config

```diff
@@ -0,0 +1,9 @@
+{plugins, [
+    couch_db_epi,
+    chttpd_epi,
+    couch_index_epi,
+    global_changes_epi,
+    mango_epi,
+    mem3_epi,
+    setup_epi
```

Comment:
This list seems too similar to the one in rel/files/sys.config. Any difference between or possibility to have/maintain only one of them?